README.md

```diff
@@ -74,26 +74,57 @@ data and restart them (if they are running).
 fluo-dev reset all
 ```
 
-Verify that your `fluo.properties` file generated by `fluo configure` is correct
+Confirm that everything started by checking the monitoring pages of Hadoop & Accumulo:
+ * [Hadoop NameNode](http://localhost:50070/)
+ * [Hadoop ResourceManager](http://localhost:8088/)
+ * [Accumulo Monitor](http://localhost:50095/)
+
+Running Fluo
+------------
+
+With its dependencies running, Fluo can now be started.  Before starting Fluo, you can optionally
+verify or change configuration in the `fluo.properties` file generated by `fluo configure`.  
 ```
 vim conf/fluo/fluo.properties
 ```
 
-Finally, deploy Fluo which will remove any existing install, rebuild fluo, install fluo, and 
-configure it using `conf/fluo/fluo.properties`.
+If you want to run Fluo with observers, avoid configuring them in `fluo.properites`.  Instead, you
+should create a file called `observer.props` in `conf/` by copying the example:
+```
+cp conf/observer.props.example conf/observer.props
```

Comment:
Once we have the quick start example working w/ beta, we may want to consider making it the example.